src/zenml/zen_server/deploy/helm/templates/server-db-job.yaml

```diff
@@ -104,7 +104,7 @@ spec:
           envFrom:
           - secretRef:
               name: {{ include "zenml.fullname" . }}-db-migration
-          {{- with .Values.resources }}
+          {{- with .Values.resourcesMigrationPod }}
```

Comment:
can you use `.Values.resources` if not set or empty ? and update the comment in values.yaml to document this please ?

Reply:
makes sense, done